docs/how-to/enforce-permissions/bulk-check.mdx

```diff
@@ -64,8 +64,9 @@ func main() {
 
 :::note
 
-Please note that the "Bulk Check" feature might not work correctly if you're using [PDP Sharding](/how-to/manage-data/pdp-sharding).
-This is because it only runs on one PDP instance at a time, and that instance only contains data for a few tenants,
-not all of them in the setup.
+Please note that the "Bulk Check" feature might have degraded performance if you're
+providing requests on different tenants. This is because we split the requests by tenant in order for the request
```

Comment:
Please note that the "Bulk Check" feature might have degraded performance if you're
checking permissions for different tenants. This is because we split the requests by tenant in order for the request
to reach the relevant PDP instance. This is done for it to be compatible with the current [PDP Sharding](/how-to/manage-data/pdp-sharding) setup, in which that instance only contains data for a few tenants, not all of them.

Reply:
fixed